.vib/jaeger/cypress/cypress/integration/jaeger_spec.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
```

Comment:
Please, configure your formatter to indent the file using 2 spaces instead of 4 to align with the rest of the tests files

Reply:
Fixed [11a5162](https://github.com/bitnami/charts/pull/13480/commits/11a5162c49b538edbd12d9d22baaea36c5212e04)